channeld/channeld.c

```diff
@@ -2915,6 +2915,38 @@ static size_t calc_weight(enum tx_role role, const struct wally_psbt *psbt)
 	return weight;
 }
 
+/* Get the fundee amount in the channel after the splice */
+static struct amount_msat splice_remote_balance(struct peer *peer,
```

Comment:
is this function name no longer accurate?

Reply:
Good point, lets try to call it `relative_splice_balance_fundee`? so in this way we can have also the dor open for `relative_splice_balance_fuder` if we need it in the future.

What do you think?